### DISCUSSION TITLE
Using Ice with CMake

### DISCUSSION THREAD

**Discussion by ethindp** [zeroc-ice/ice#1434] [Q&A]
I can't seem to find any documentation on how to actually get this to work. There is of course the [FindIce.CMake](https://cmake.org/cmake/help/latest/module/FindIce.html) documentation, and there's [this repository](https://github.com/ome/openmicroscopy), but I was hoping for a slimmed-down or easier to consume tutorial that gae me a bare-bones example I could look at for consuming Ice in CMake projects and generating source code from Slice definitions. Is anyone interested in this kind of work, given that CMake is (most likely) the most popular build system for C/C++ projects, and given that it's also the most portable? I'll wade through a 500-line CMake file if I must but I'd rather not if there's something I'm just missing.

**Comment by andrejstrogonov** [comment]
You can use bastard, but it`s unsupported project. Of course, I want to see CMake configuration file because Windows way is non trivial way. Unfortunately CI/CD is so bad:(
